xyz

```diff
@@ -157,17 +158,12 @@ for script in "${scripts[@]}" ; do
   run "VERSION=$next_version PREVIOUS_VERSION=$version '$script'"
 done
 
-inc() {
-  if [[ -f "$1" ]] ; then
-    run "node -e '$(echo "
-      var o = require('./$1'); o.version = '$next_version';
-      require('fs').writeFileSync('./$1', JSON.stringify(o, null, 2) + '\n');
```

Comment:
```bash
      require('fs').writeFileSync('./$1', JSON.stringify(o, null, 2) + '\n');
                                       ^-- SC2028
```

[SC2028][1]: `echo` won't expand escape sequences. Consider `printf`.

_I think this should point to `\n`._


[1]: https://github.com/koalaman/shellcheck/wiki/SC2028

Reply:
But you're not using `echo` here anymore, right?

Reply:
Correct. The most elegant fix involved removing the `inc` function and the convoluted `tr "'" '"' | tr -d "\n" | tr -s " " | sed -e "s/^ *//" -e "s/ *$//"` pipeline. The code for updating the `version` field of __package.json__ will become even nice in another pull request; this is a step in the right direction.